src/crimson/osd/watch.cc

```diff
@@ -102,6 +103,18 @@ seastar::future<> Watch::remove(const bool send_disconnect)
     });
 }
 
+bool notify_reply_t::operator<(const notify_reply_t& rhs) const
```

Comment:
could you squash this change into "crimson/osd: implement the complete phase of Notify propagation." ?